<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Symfony\Component\Process\Exception\InvalidArgumentException" namespace="Symfony\Component\Process\Exception" name="InvalidArgumentException" abstract="false" final="false" start="19" end="21">
  <file path="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/symfony/process/Exception" file="InvalidArgumentException.php" realpath="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/symfony/process/Exception/InvalidArgumentException.php" size="496" time="2015-12-17T18:44:10+00:00" unixtime="1450377850" sha1="1afc135760927a15f53345e49d763fef6cea2098" relative="vendor/symfony/process/Exception/InvalidArgumentException.php"/>
  <docblock>
    <description compact="InvalidArgumentException for the Process Component."/>
    <author value="Romain Neutron &lt;imprec@gmail.com&gt;"/>
  </docblock>
  <extends full="InvalidArgumentException" namespace="" name="InvalidArgumentException"/>
  <implements full="Symfony\Component\Process\Exception\ExceptionInterface" namespace="Symfony\Component\Process\Exception" name="ExceptionInterface"/>
</class>
